.github/workflows/go.yml

```
@@ -0,0 +1,103 @@
+
+    - name: Install dependencies
+      run: cd ts-wrapper && npm ci
+
+    - name: Lint
+      run: cd ts-wrapper && npm run lint
+
+  ts-build:
+    name: TypeScript Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)